addons/website_sale/models/sale_order.py

```diff
@@ -80,6 +80,13 @@ def _compute_abandoned_cart(self):
             else:
                 order.is_abandoned_cart = False
 
+    def _compute_require_signature(self):
+        for order in self:
+            if order.website_id:
+                order.require_signature = False
+            else:
+                super()._compute_require_signature()
```

Comment:
```suggestion
        website_orders = self.filtered('website_id')
        website_order.require_signature = False
        super(SaleOrder, self-website_orders)._compute_require_signature()
```

Please batch operations when possible ;)